lib/gym/build_command_generator.rb

```diff
@@ -40,7 +40,7 @@ def options
         options << "-sdk '#{config[:sdk]}'" if config[:sdk]
         options << "-destination '#{config[:destination]}'" if config[:destination]
         options << "-archivePath '#{archive_path}'"
-        options << "-xcargs '#{config[:xcargs]}'" if config[:xcargs]
+        options << config[:xcargs] if config[:xcargs]
```

Comment:
When passing multiple key-value pairs, putting quotes around them confuses xcodebuild. For example "A=1 B=2" gets interpreted as a key called "A" with a value "1 B=2" I believe. So I decided just to match Shenzhen's behavior here.